### PR TITLE
Replace playground with model_servers

### PR DESCRIPTION
**Pull request by MichaelClifford**
Partially Address #89 (as this is a large change, planning on breaking it up into a couple of PRs)

* Moves all files from `playground/` into `model_servers/llamacpp_python/`
* Updated ai-lab.yamls to point to the new file locations.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
But you need to sign your commits.

**Review by lstocchi — COMMENTED** [review]
Forgot to update https://github.com/containers/ai-lab-recipes/blob/main/image_understanding/ai-lab.yaml ?

**Comment by MichaelClifford** [comment]
Thanks @lstocchi added.

File: chatbot/ai-lab.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ application:
   description: This is a Streamlit chat demo application. 
   containers:
     - name: llamacpp-server
-      contextdir: ../playground
-      containerfile: Containerfile
+      contextdir: ../model_servers/llamacpp_python
+      containerfile: ../model_servers/llamacpp_python/base/Containerfile
```
Comment (lstocchi) [inline chatbot/ai-lab.yaml]: I didn't notice before but i think that the containerfile should be `containerfile: ./base/Containerfile`

Comment (lstocchi) [reply chatbot/ai-lab.yaml]: This is related to all ai-lab files

Comment (MichaelClifford) [reply chatbot/ai-lab.yaml]: Ok, was was actually not sure. It will start from the contextDir?

Comment (lstocchi) [reply chatbot/ai-lab.yaml]: yes

File: summarizer-langchain/ai-lab.yaml [hunk 2]
```diff
@@ -5,8 +5,8 @@ application:
   description: This is a Streamlit demo application for summarizing text. 
   containers:
     - name: llamacpp-server
-      contextdir: ../playground
-      containerfile: Containerfile
+      contextdir: ../model_servers/llamacpp_python
+      containerfile: ./model_servers/llamacpp_python/base/Containerfile
```
Comment (sallyom) [inline summarizer-langchain/ai-lab.yaml]: this needs to be ./base/Containerfile, right?

**Review by sallyom — APPROVED** [review]
LGTM!

**Pull request merged**